README.rst

```diff
@@ -127,7 +127,7 @@ Spread the word, give feedback, help with documentation, testing or development.
 
 You can also give monetary support to the project, see there for details:
 
-https://borgbackup.readthedocs.io/en/stable/support.html#bounties-and-fundraisers
+https://www.borgbackup.org/support/support.html#bounties-and-fundraisers
```

Comment:
hmm, wasn't it support/free.html (vs. commercial.html) in the end?

Reply:
that it was. Gonna close this and re-do when fixes are merged.

Reply:
hmm, this is still todo, right?

Reply:
Nope, complete with my other backport PR